composer.json

```diff
@@ -24,19 +24,11 @@
       }
     ],
     "require": {
-        "wp-coding-standards/wpcs": "2.1.1",
+        "wp-coding-standards/wpcs": "~2.3.0",
```

Comment:
This will allow e.g. 2.3.1 2.3.2 2.3.3... go get updated automatically. So if something breaks in wpcs it can get fixed quicker vs me having to bump the PATCH version everytime.